### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-dialog-behavior#39]
On Android with Talkback, when I open a dialog, it's nice that the title is read aloud. However, when I swipe to try to read the content, I just go to the next button and skip over the dialog content. Focus should remain in the dialog until the user jumps out of it. This seems to only be the case for the dialog button. The Modal dialog and scrolling dialog seem to be working as expected.

**Comment by valdrinkoshi** [comment]
We should update the demo page to use make the content of dialogs readable (e.g. use `h2` so that `aria-labelledby` gets set)

**Comment by valdrinkoshi** [comment]
First of all, Talkback will read any element (also the ones that cannot be reached via Tab on desktop, e.g. `<p>, <div>, <br>` - yes, `<br>` http://jsbin.com/nipoci/2/edit?html,output)

Also, Talkback doesn't handle `aria-labelledby` and `tabindex` together: http://jsbin.com/bagebu/2/edit?html,output
- 1st div contents can be reached through swiping and are read correctly
- when reaching the 2nd div, I hear only `title 2` and cannot reach the paragraph

**Comment by valdrinkoshi** [comment]
Tested on mac VoiceOver, and the paragraph is not read on the second div.
I guess the solution is removing the setting of `aria-labelledby`, which should not be forced by dialogs, and delegate its setting to the user of the element.
